planning/behavior_path_planner/src/utils/lane_change/utils.cpp

```diff
@@ -228,14 +228,18 @@ bool isPathInLanelets(
 }
 
 std::optional<LaneChangePath> constructCandidatePath(
-  const PathWithLaneId & prepare_segment, const PathWithLaneId & target_segment,
-  const PathWithLaneId & target_lane_reference_path, const ShiftLine & shift_line,
-  const lanelet::ConstLanelets & original_lanelets, const lanelet::ConstLanelets & target_lanelets,
-  const std::vector<std::vector<int64_t>> & sorted_lane_ids,
-  const LaneChangePhaseInfo longitudinal_acceleration, const double lateral_acceleration,
-  const LaneChangePhaseInfo lane_change_length, const LaneChangePhaseInfo lane_change_velocity,
-  const double terminal_lane_changing_velocity, const LaneChangePhaseInfo lane_change_time)
+  const LaneChangeInfo & lane_change_info, const PathWithLaneId & prepare_segment,
+  const PathWithLaneId & target_segment, const PathWithLaneId & target_lane_reference_path,
+  const std::vector<std::vector<int64_t>> & sorted_lane_ids)
 {
+  const auto & shift_line = lane_change_info.shift_line;
+  const auto & original_lanelets = lane_change_info.reference_lanelets;
+  const auto & target_lanelets = lane_change_info.target_lanelets;
+  const auto & terminal_lane_changing_velocity = lane_change_info.terminal_lane_changing_velocity;
+  const auto & longitudinal_acceleration = lane_change_info.longitudinal_acceleration;
+  const auto & lane_change_velocity = lane_change_info.velocity;
+  const auto & lane_change_length = lane_change_info.length;
```

Comment:
```suggestion
  const auto terminal_lane_changing_velocity = lane_change_info.terminal_lane_changing_velocity;
  const auto longitudinal_acceleration = lane_change_info.longitudinal_acceleration;
  const auto lane_change_velocity = lane_change_info.velocity;
  const auto lane_change_length = lane_change_info.length;
```

Actually for plain types, normally `&` is not included.